labs/aries-fabric.md

```diff
@@ -0,0 +1,29 @@
+# Aries Wrapper For Fabric
+
+# Description
+
+Self - Sovereign Identity application is an application by which you are able to keep your identity secured and at the same time also prove your identity to the verifier without revealing them. These applications are built using Hyperledger Aries, which depends on Indy as a Distributed Ledger and Ursa as a cryptography library. Currently, the most used Blockchain platform is Hyperledger Fabric. And the businesses that are building solutions using Fabric want to have some sort of identity verification to be done in their platform. This is the reason, why this project has come into existence. Once done it would enable Aries to interact with Fabric as a ledger, which would then enable SSI applications to be built on top of Fabric.
+
+# Scope of Lab
+
+As Hyperledger's mission is to increase the adoption of Blockchain technologies for business, this project would enable business that are using Fabric for building their blockchain solutions to have identity verification process to be in-built in their system. The future developements that it would undergo once it is set as a project in lab are :
+
+Currently AFJ only supports did, schema and credential definition as domain transactions. Once revocation and attrib transactions are added, we would add support for those as well in the chaincode and module.
+
+Now there are 2 separate wallets – indy wallet for agent and fabric wallet for client application, we would try to integrate that into one.
+
+We would also try to add support such that AFJ works with other ledger as well.
+
+# Initial Committers
+
+https://github.com/HarshMultani
+
+# Sponsor
+
+Kamlesh Nagware ( https://github.com/knagware9 ) - Role ( TSC Member )
+
+# Pre-existing repository
+
+https://github.com/HarshMultani/aries-fabric-wrapper
+
+https://github.com/HarshMultani/aries-framework-javascript
```

Comment:
This is forked from hyperledger/aries-framework-javascript. I do not think we should fork this to the hyperledger-labs organization, but rather, have you work on a branch of hyperledger-aries-framework for this work.

Reply:
@HarshMultani : thoughts on ^this?

Reply:
I agree with @tkuhrt that this is a bit of an issue. @HarshMultani have you ask the maintainers of the original repo whether they'd let you have you work on a branch of the main repo?

Reply:
Hi @tkuhrt , @lehors , I agree with you on this. Let me check with the maintainers of AFJ.

Reply:
So the plan is to just migrate `https://github.com/HarshMultani/aries-fabric-wrapper` to the hyperledger-labs organization. The other will remain in the AFJ repo. Correct?

Reply:
Yes.